source/release-notes.txt

```diff
@@ -20,6 +20,18 @@ Release Notes for |bi|
 .. expect and we don't break the links on the downloads page.
 .. (DOCS-9670 for context)
 
+.. _bi-2-14-6:
+
+|bi| 2.14.6
+-----------
+
+*Released May 17th, 2023*
+
+Improvements
+~~~~~~~~~~~~
+
+- Now built with Go 1.19.0.
```

Comment:
[[BI-2755](https://jira.mongodb.org/browse/BI-2755)] - `3rd party Critical/High tracking and remediation`:
This was done as part as the Go 1.19 upgrade, but we decided to create a separate ticket to make it clear that we updated libraries with Critical/High vulnerabilities.
It could be worth mentioning as well.

Reply:
@lmkerbey-mdb This seems pretty important for Salman Baset on the Product team as the IBM contract depends on it. Can you please add a mention of it, or if this is not something we put in release notes, can you let me know.